user-interface/src/main/java/life/qbic/datamanager/MyVaadinSessionInitListener.java

```diff
@@ -34,12 +38,23 @@ public void serviceInit(ServiceInitEvent event) {
     event.getSource().addSessionInitListener(
         initEvent -> log.info("A new Session has been initialized!"));
 
+    event.getSource().addSessionDestroyListener(this);
+
     event.getSource().addUIInitListener(
         initEvent -> {
           log.info("A new UI has been initialized!");
           UI ui = initEvent.getUI();
           ui.getPage().retrieveExtendedClientDetails(clientDetailsReceiver);
           ui.getSession().setErrorHandler(errorEvent -> uiExceptionHandler.error(errorEvent, ui));
         });
+
+
+  }
+
+  @Override
+  public void sessionDestroy(SessionDestroyEvent event) {
+    log.debug("Session destroyed.");
```

Comment:
can we log some kind of session id here? I imagine this information is not very usable if multiple sessions exist at the same time and we cannot differentiate.